### DISCUSSION TITLE
ADMIN_TOKEN is set but still no admin page.

### DISCUSSION THREAD

**Discussion by mciaio** [dani-garcia/vaultwarden#3746] [Q&A] [answered]
Running a docker container in Portainer.  Updated the docker compose to include ADMIN_TOKEN=something but still getting the error "The admin panel is disabled, please configure the 'ADMIN_TOKEN' variable to enable it"
![compose](https://github.com/dani-garcia/vaultwarden/assets/25258111/f11da6ce-3c8f-486c-a7ad-56025a3d87be)
![Log](https://github.com/dani-garcia/vaultwarden/assets/25258111/681416c7-dad5-4311-ade9-a702fcbe65db)

I do not have a config.json file in the data directory.  Any help would be appreciated.

**Comment by kokofixcomputers** [comment, marked as answer]
Can you try restarting the server? It works for me.

**Comment by Masgalor** [comment]
You can use `docker exec vaultwarden printenv` to check if the environment variable is applied to your container.
